cli: Include route and route rule when `nmstatectl show iface`

cathay4t opened the pull request:
Include route and route rule for specified interface of command
`nmstate show <iface_name>`.

Integration test case included.

cathay4t commented:
The tier 1 CI failure on nm-main is caused by regression of NetworkManager: https://bugzilla.redhat.com/show_bug.cgi?id=2076131#c14

cathay4t commented:
The tier 2 nm-main failure on `test_disable_vlan_filtering_on_bridge` is due to NM bug: https://bugzilla.redhat.com/show_bug.cgi?id=2092762

ffmancera commented on tests/integration/nmstatectl_test.py:
```diff
@@ -282,3 +287,87 @@ def assert_command(cmd, expected_rc=cmdlib.RC_SUCCESS):
     return ret
+
+
+@pytest.fixture
+def eth1_with_static_route_and_rule(eth1_up):
```
What about making this tier1 test? It should be really stable

cathay4t replied:
Done.

cathay4t committed:
cli: Include route and route rule when `nmstatectl show iface`

Include route and route rule for specified interface of command
`nmstate show <iface_name>`.

Integration test case included.

Signed-off-by: Gris Ge <fge@redhat.com>
